### PR TITLE
Section 1: Motivation and Purpose

### LINKED ISSUE #2: Milestone1 task checklist
Submission
- [x] repolink + team contract [@zjr-mds](https://ubc-mds.slack.com/team/U02DJ56H6RE) https://docs.google.com/document/d/1ANXtaNkJctWvbY645RifauygL7fTx1bi7hMorufJwOg/edit

Setup:
- [x] Create repo structure [@adrianne-l ]

Proposal:
- [x] section 1: motivation and purpose; North America audience [@zackt113 ]
- [x] section 2: Data description and mentioning about the filtered data (only the ones after 1980)  [@lipcai ]
       https://github.com/vega/vega-datasets/blob/next/data/movies.json 
 - [x] section 3: research question and usage scenarios: using `US-gross revenue`, `runtime`, and `rating` to find the relationship between ratings and runtime, top-rated movies based on year or genre, and revenue with genres [@adrianne-l ]

Description:
- [x] app sketch + description:  3 visulizations with bar + scatter plot + table of top-rated movies(ranking, titles, rating(or duration), MPAA Rating) [@zjr-mds]

### PR DESCRIPTION
**Pull request by zackt113**

### CODE REVIEW

File: docs/proposal.md [hunk 1]
```diff
@@ -2,7 +2,17 @@
...
 ## Motivation and Purpose
 
-TBC
+Our role: Data Analysts at a consulting firm
+
+Target Audience: Movie enthusiasts & Directors at Movie Production Companies
+
+Movie enthusiasts:
+
+Millions of movies are published on a variety of channels; however, filtering excellent movies could be time-consuming. If we could provide a list of popular movies based on customers' preference in genres and productions years, they will save a large amount of time on searching. To solve this challenge, we propose building a data visualization app that allows movie enthusiasts to visually present the top 10 ranking movies in a given year range or genre. Our app will visualize the selection process by providing a slider for the year range and dropdowns for genres. Eventually, detailed information (ranking, title, duration, MPAA ratings) of the top 10 movies will be rendered in a table format.
```
Comment (zjr-mds) [inline docs/proposal.md]: Great work! Only have one suggestion on the last few sentences, we could make it less technical and mention about something like this app can be served as a great exploratory tool for people who like visualization and friendly-user-interface to extract various data at the same time with minimum efforts to select the filters.

"
...to visually present the top 10 ranking movies in a given year range or genre. Our app will visualize the selection process by providing a slider for the year range and dropdowns for genres. Eventually, detailed information (ranking, title, duration, MPAA ratings) of the top 10 movies will be rendered in a table format. "

**Review by zjr-mds — CHANGES_REQUESTED** [review]

File: docs/proposal.md (hunk 1 above)
Comment (zackt113) [reply docs/proposal.md]: Thanks, I have incorporated your suggestion into the proposal!

File: docs/proposal.md [hunk 2]
```diff
@@ -4,15 +4,9 @@
 
 Our role: Data Analysts at a consulting firm
 
-Target Audience: Movie enthusiasts & Directors at Movie Production Companies
+Target Audience: Movie enthusiasts who enjoy researching movies
```
Comment (zjr-mds) [inline docs/proposal.md]: not sure if keeping this criterion less specific would be better? Hope to get a second opinion!
```suggestion
Target Audience: Movie enthusiasts 
```

**Review by zjr-mds — APPROVED** [review]
Looking good to me! @adrianne-l feel free to merge it if this looks good to you, thank you!

**Review by adrianne-l — APPROVED** [review]
I agree on the changes. No further comments.

**Pull request merged**